react/features/filmstrip/subscriber.web.js

```diff
@@ -123,3 +124,12 @@ StateListenerRegistry.register(
             );
         }
     });
+
+/**
+ * Listens for changes in the client width to determine whether the overflow menu(s) should be displayed as drawers.
+ */
+StateListenerRegistry.register(
+    /* selector */ state => state['features/base/responsive-ui'].clientWidth < 500,
```

Comment:
Isn't this 500 defined somewhere as a constant? We should use it consistently in case we ever decide to change it to a different value like 512, which is a power of 2, so a much better choice!